Bld/build.sh

```diff
@@ -5,16 +5,16 @@ GREEN='\033[0;32m'
 ORANGE='\033[0;33m'
 set -e
 
+BLD_PATH=$(dirname $0)
+pushd $BLD_PATH/..
+
 echo -e "${ORANGE} ---- Fetching git submodules ----${NOCOLOR}"
-pushd ..
+
 # Initialize submodules
```

Comment:
Thanks a lot, I was lazy to never fix this.

Reply:
Yeah, I just ended up getting into the habit of calling `dotnet` directly instead :)